### PR TITLE
[Bot] Update docs for latest fastlane release (actions.md, available-plugins.md) 🚀

### PR DESCRIPTION
**Pull request by joshdholtz**
Auto-generated by _fastlane_ bot 🤖

### CODE REVIEW

File: docs/plugins/available-plugins.md [hunk 1]
```diff
@@ -449,17 +449,17 @@ tests | <b>24</b> | The more tests a plugin has, the better
...
-  **suppress_tests_from_junit** | testing | Uses a junit xml report file to suppress either passing or failing tests in an Xcode Scheme
-  **tests_from_xctestrun** | testing | Retrieves all of the tests from xctest bundles referenced by the xctestrun file
-  **quit_core_simulator_service** | - | Force-quits the com.apple.CoreSimulator.CoreSimulatorService.
-  **collate_test_result_bundles** | testing | Combines multiple test_result bundles into one test_result bundle
-  **tests_from_junit** | testing | Retrieves the failing and passing tests as reported in a junit xml file
-  **multi_scan** | testing | Uses scan to run Xcode tests a given number of times, with the option of batching and/or parallelizing them, only re-testing failing tests.
-  **collate_html_reports** | testing | Combines multiple html report files into one html report file
-  **collate_junit_reports** | testing | Combines multiple junit report files into one junit report file
-  **collate_json_reports** | testing | Combines multiple json report files into one json report file
+  **suppressed_tests** | testing | -
```
Comment (lyndsey-ferguson) [inline docs/plugins/available-plugins.md]: The change to read the descriptions of plugin actions seems to be broken? I know that this is definitely not the description I've provided to my plugin's actions 😄 

For example, [`suppressed_tests`](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/lib/fastlane/plugin/test_center/actions/suppressed_tests.rb#L57) is:

> Retrieves a list of tests that are suppressed in a specific or all Xcode Schemes in a project
Not `-`

Comment (janpio) [reply docs/plugins/available-plugins.md]: Ping @mollyIV

Comment (lyndsey-ferguson) [reply docs/plugins/available-plugins.md]: # 👏

Comment (mollyIV) [reply docs/plugins/available-plugins.md]: Most probably the regression from https://github.com/fastlane/fastlane/pull/15212 😿 
Thanks for spotting that, will have a look 👍

Comment (mollyIV) [reply docs/plugins/available-plugins.md]: Created the PR which reverts the regression changes: https://github.com/fastlane/fastlane/pull/15255 to not block future `fastlane` releases and being able to update the docs with each release.

Comment (janpio) [reply docs/plugins/available-plugins.md]: IMO the docs are never blocking, the releases go out anyway - so feel free to just find a fix for this whenever you have time @mollyIV and we will merge that one asap.

Comment (mollyIV) [reply docs/plugins/available-plugins.md]: Created a fix for single line action's description rendering: https://github.com/fastlane/fastlane/pull/15266 🙇 

I think we should close this PR and open a new one once the fix is merged. 
CC: @joshdholtz

**Pull request closed without merging**